blog/releases/announcing-knative-v0-19-release.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Version v0.19 release"
-linkTitle: "Version v0.19 release"
+title: "Version 0.19 release"
```

Comment:
see above and https://docs.google.com/spreadsheets/d/1p1_kBUd6ZvonxHkMcEJPayf6QIpExuFf5cFq0ptar7I/edit#gid=0